src/helm/config/model_metadata.yaml

```diff
@@ -1729,6 +1729,22 @@ models:
     release_date: 2024-02-05
     tags: [TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG]
 
+  - name: Qwen/Qwen-VL
```

Comment:
Let's stay consistent and name this with lower case letters like we did for `qwen/qwen1.5-72b`